owl/sys.scm

```diff
@@ -135,7 +136,10 @@
       
       (define (mem-strings ptr)
          (mem-array-map ptr mem-string))
-      
+
+      (define (errno)
```

Comment:
It would now also be useful to have strerror() -> char *, which could be read with mem-string.

Reply:
Yes, good idea! Will do that after some changes I have in the queue.